map_content.txt

```diff
@@ -1139,3 +1139,98 @@
 966 13	doors:door_steel_a 1
 966 14	doors:door_steel_a 2
 966 15	doors:door_steel_a 3
+
+
+//spruce wood door
+968 0	doors:door_wood_a 1
+968 1	doors:door_wood_a 2
+968 2	doors:door_wood_a 3
+968 3	doors:door_wood_a 0
+968 4	doors:door_wood_b 2
+968 5	doors:door_wood_b 3
+968 6	doors:door_wood_b 0
+968 7	doors:door_wood_b 1
+968 8	doors:door_wood_b 1
+968 9	doors:door_wood_b 2
+968 10	doors:door_wood_b 3
+968 11	doors:door_wood_b 0
+968 12	doors:door_wood_a 0
+968 13	doors:door_wood_a 1
+968 14	doors:door_wood_a 2
+968 15	doors:door_wood_a 3
+
+
+//birch wood door
+970 0	doors:door_wood_a 1
+970 1	doors:door_wood_a 2
+970 2	doors:door_wood_a 3
+970 3	doors:door_wood_a 0
+970 4	doors:door_wood_b 2
+970 5	doors:door_wood_b 3
+970 6	doors:door_wood_b 0
+970 7	doors:door_wood_b 1
+970 8	doors:door_wood_b 1
+970 9	doors:door_wood_b 2
+970 10	doors:door_wood_b 3
+970 11	doors:door_wood_b 0
+970 12	doors:door_wood_a 0
+970 13	doors:door_wood_a 1
+970 14	doors:door_wood_a 2
+970 15	doors:door_wood_a 3
+
+
+//jungle wood door
+972 0	doors:door_wood_a 1
+972 1	doors:door_wood_a 2
+972 2	doors:door_wood_a 3
+972 3	doors:door_wood_a 0
+972 4	doors:door_wood_b 2
+972 5	doors:door_wood_b 3
+972 6	doors:door_wood_b 0
+972 7	doors:door_wood_b 1
+972 8	doors:door_wood_b 1
+972 9	doors:door_wood_b 2
+972 10	doors:door_wood_b 3
+972 11	doors:door_wood_b 0
+972 12	doors:door_wood_a 0
+972 13	doors:door_wood_a 1
+972 14	doors:door_wood_a 2
+972 15	doors:door_wood_a 3
+
+
+//acacia wood door
+974 0	doors:door_wood_a 1
+974 1	doors:door_wood_a 2
+974 2	doors:door_wood_a 3
+974 3	doors:door_wood_a 0
+974 4	doors:door_wood_b 2
+974 5	doors:door_wood_b 3
+974 6	doors:door_wood_b 0
+974 7	doors:door_wood_b 1
+974 8	doors:door_wood_b 1
+974 9	doors:door_wood_b 2
+974 10	doors:door_wood_b 3
+974 11	doors:door_wood_b 0
+974 12	doors:door_wood_a 0
+974 13	doors:door_wood_a 1
+974 14	doors:door_wood_a 2
+974 15	doors:door_wood_a 3
+
+
+//dark oak wood door
+976 0	doors:door_wood_a 1
+976 1	doors:door_wood_a 2
+976 2	doors:door_wood_a 3
+976 3	doors:door_wood_a 0
+976 4	doors:door_wood_b 2
+976 5	doors:door_wood_b 3
+976 6	doors:door_wood_b 0
+976 7	doors:door_wood_b 1
+976 8	doors:door_wood_b 1
+976 9	doors:door_wood_b 2
+976 10	doors:door_wood_b 3
+976 11	doors:door_wood_b 0
+976 12	doors:door_wood_a 0
+976 13	doors:door_wood_a 1
+976 14	doors:door_wood_a 2
+976 15	doors:door_wood_a 3
```

Comment:
I'm assuming the in-game ABM still converts this all ok? Did you test that?

Reply:
Like the currently converted wood/steel doors. I mapped only the bottom halves(from Minecraft) to the full MTG doors. Yes, I tested; they convert fine.